packages/suite/src/components/suite/PinInput/components/InputPin/index.tsx

```diff
@@ -3,15 +3,17 @@ import styled from 'styled-components';
 import { variables, useTheme, Icon, Input } from '@trezor/components';
 
 const StyledInput = styled(Input)`
-    letter-spacing: 8px;
-    min-width: 256px;
-    height: 54px;
-    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
-    font-size: ${variables.FONT_SIZE.SMALL};
-    padding: 0 32px 0 20px;
-    color: ${({ theme }) => theme.TYPE_DARK_GREY};
-    background: transparent;
-    box-sizing: border-box;
+    input {
+        letter-spacing: 8px;
```

Comment:
Can be used from `spacingsPx`

Reply:
cdd1b94ab

But, please, do not concentrate on it outside of the `components` package for now 🙌